packaging/windows/WIXInstallers/KeybaseBundle/Bundle.wxs

```diff
@@ -114,7 +114,9 @@
                   SourceFile="Redist\VCRedist_2015\vc_redist.x86.exe"
                   DownloadUrl="https://download.microsoft.com/download/2/6/8/268f7be7-8d2a-42d6-a823-5c6941a3d70a/vc_redist.x86.exe"
       >
-        <ExitCode Behavior="scheduleReboot"/>
+        <ExitCode Value="1641" Behavior="scheduleReboot"/>
+        <ExitCode Value="3010" Behavior="scheduleReboot"/>
+        <ExitCode Value="0" Behavior="success" />
```

Comment:
Is there any worry that there might be some more obscure exit codes that also require rebooting? Maybe things we were handling before with our aggressive reboots, that could slip through now?